Examples/Algorithms/Digitization/include/ActsExamples/Digitization/DigitizationConfig.hpp

```diff
@@ -121,17 +116,15 @@ struct DigiComponentsConfig {
 
 class DigitizationConfig {
```

Comment:
I would also like the digitizations algorithm to follow our usual idiom to be a sub-struct to the algorithm class...

Reply:
I agree - I was just about to move it but there seems to be a lot of stuff in the file. Do you see a good way to move this?

Reply:
Hmm so I think there is an asymetry here: The config of the smeared digi component has a dedicated file (`SmearingConfig.hpp`), the geometric one is thrown together with the algorithm config.
I could immagine we could rename this file in `GeometricConfig.hpp`, and move the algorithm config to the algorithm?